.github/workflows/tag.yml

```diff
@@ -62,15 +62,15 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
-    - name: Install bumpversion
+    - name: Install bump2version
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install bumpversion
-    - name: Run bumpversion ${{ env['BV_PART'] }}
+        python -m pip install "bump2version~=1.0"
+    - name: Run bump2version ${{ env['BV_PART'] }}
       run: |
         OLD_TAG=$(git describe --tags --abbrev=0)
         echo "::set-env name=OLD_TAG::${OLD_TAG}"
-        bumpversion $BV_PART --message "Bump version: {current_version} → {new_version}
+        bump2version $BV_PART --message "Bump version: {current_version} → {new_version}
```

Comment:
There are other instances of the word `bumpversion` in this workflow, but these are the only ones that matter as the rest are related to the tag names on GitHub.